pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go

```diff
@@ -181,6 +206,54 @@ func (c *APIReconciler) enqueueNegotiatedAPIResource(obj interface{}) {
 	}
 }
 
+func apiExportGroupResources(apiExport *apisv1alpha1.APIExport) map[schema.GroupResource]interface{} {
+	grs := map[schema.GroupResource]interface{}{}
+	for _, resourceSchema := range apiExport.Spec.LatestResourceSchemas {
+		parts := strings.SplitN(resourceSchema, ".", 3)
+		if len(parts) != 3 {
+			klog.Errorf("invalid LatestResourceSchemas list element on APIExport %s|%s: %s", logicalcluster.From(apiExport), apiExport.Name, resourceSchema)
```

Comment:
continue?

Reply:
Oh, yes, sure !

Reply:
let me rebase and force-push for this fix.

Reply:
Done